code/index.html

```diff
@@ -26,14 +25,27 @@ <h1>Does the person have</h1>
         <optgroup label="hair">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="black">black hair</option>
+          <option value="hidden">hidden hair</option>
+          <option value="purple">purple hair</option>
+          <option value="orange">orange hair</option>
+          <option value="gray">gray hair</option>
+          <option value="white">white hair</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="green">green eyes</option>
+          <option value="hidden">brown eyes</option>
+     
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
+          <option value="hat">Hat</option>
           <option value="glasses">glasses</option>
+          
+
           <!-- Add the other accessory option here. (hat)  -->
         </optgroup>
         <optgroup label="other">
```

Comment:
change this to:

<option value="smoking habit">Smoking habit</option>

in order for the alert messages to be correct.